scripts/setup/create_singlenode_container.sh

```diff
@@ -37,14 +37,19 @@ server_exec() {
     # Spin up vHive under container mode.
     server_exec 'sudo DEBIAN_FRONTEND=noninteractive apt-get autoremove' 
     server_exec "git clone --branch=$VHIVE_BRANCH https://github.com/ease-lab/vhive"
-    server_exec 'cd vhive; ./scripts/cloudlab/setup_node.sh stock-only'
+    
+    server_exec 'sudo wget -q https://github.com/mikefarah/yq/releases/download/v4.27.3/yq_linux_amd64 -O /usr/bin/yq && sudo chmod +x /usr/bin/yq'
+    server_exec 'bash -s' < ~/invitro/scripts/setup/rewrite_yaml_files.sh
+
+    server_exec "pushd ~/vhive/scripts > /dev/null && ~/vhive/scripts/install_go.sh && source /etc/profile && go build -o setup_tool && ~/vhive/scripts/setup_tool setup_node ${OPERATION_MODE} && popd > /dev/null"
+    
     server_exec 'tmux new -s runner -d'
     server_exec 'tmux new -s kwatch -d'
     server_exec 'tmux new -d -s containerd'
     server_exec 'tmux new -d -s cluster'
     server_exec 'tmux send-keys -t containerd "sudo containerd" ENTER'
     sleep 3
-    server_exec 'cd vhive; ./scripts/cluster/create_one_node_cluster.sh stock-only'
+    server_exec 'pushd ~/vhive/scripts > /dev/null && /usr/local/go/bin/go build -o setup_tool && ./setup_tool create_one_node_cluster stock-only && popd > /dev/null'
```

Comment:
Here you should already have go properly installed. No need to use absolute path to it.